v2/tests/api_test.go

```diff
@@ -4,7 +4,7 @@ import (
 	"context"
 	"testing"
 
-	"github.com/Concordium/concordium-go-sdk/v2"
+	v2 "github.com/Concordium/concordium-go-sdk/v2"
```

Comment:
Why is this needed?

Reply:
This is added automatically every time i save.